blueocean-dashboard/src/main/js/components/NotFound.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Fullscreen from './Fullscreen';
+import { Link } from 'react-router';
+
+/**
+ * Simple component to render a fullscreen 404 page
+ */
+export default () => (
+    <Fullscreen className="not-found">
+        <div className="message-box">
+            <h3>Page not found (404)</h3>
+            <div className="message">Jenkins could not find the page you were looking for. Check the URL for errors or press the back button.</div>
```

Comment:
so may be put "browser"  before the "back button" so it is clear for stupid people like me.